internal/server/v2/observation/contained_in.go

```diff
@@ -66,6 +67,43 @@ func trimDirectResp(resp *pbv2.ObservationResponse) *pbv2.ObservationResponse {
 	return result
 }
 
+// For mocking in tests.
+var (
+	getPlacesIn = placein.GetPlacesIn
+	fetchRemote = fetchRemoteWrapper
+)
+
+func fetchRemoteWrapper(
+	metadata *resource.Metadata,
+	httpClient *http.Client,
+	apiPath string,
+	remoteReq *pbv2.NodeRequest,
+) (*pbv2.NodeResponse, error) {
+	remoteResp := &pbv2.NodeResponse{}
+	return remoteResp, util.FetchRemote(metadata, httpClient, apiPath, remoteReq, remoteResp)
+}
+
+func storeFetchChildPlaces(
+	ctx context.Context,
+	store *store.Store,
+	ancestor, childType string,
+) (map[string][]string, error) {
+	return getPlacesIn(
```

Comment:
nit: no need to wrap lines here.

Reply:
Sorry - my format on save was not on. All formatting issues should be fixed now.

We should add a format checker on PRs like in the other repos.